samples/highcharts/chartchooser/continuous-distribution-scatter/demo.css

```diff
@@ -0,0 +1,8 @@
+#container {
+    max-width: 500px;
+    height: 600px;
+    position: absolute;
```

Comment:
Why absolute? This messes up the layout in the utils and potentially in other pages (as www.highcharts.com/samples) and should be avoided. Then also remove left and right. If the purpose is control over the left and right whitespace, `margin` should be used.

Reply:
Yes, the main reason is to control over the left and right.